src/services/mail.ts

```diff
@@ -0,0 +1,70 @@
+import nodemailer, { SentMessageInfo } from 'nodemailer'
+
+const MAX_TIME_RETRY_SEND_MAIL = process.env.MAX_TIME_RETRY_SEND_MAIL || 3
+export async function sendMail(
+  options: {
+    mailFrom: string
+    mailTo: string
+    mailCc?: any
+    mailBcc?: any
+    subject: string
+    fileName?: string
+    filePath?: string
+  },
+  retryCount = 0,
+): Promise<SentMessageInfo> {
+  try {
+    console.log('Start send mail to', options.mailTo)
+    let transporterOptions: any = {
```

Comment:
Prevent using `let`. In this case, we can use `const` like this:

```suggestion
    const transporterOptions = process.env.NODE_ENV === 'development' ? { /* development options */ } : { /* production options */ }
```